src/utils.ts

```diff
@@ -0,0 +1,9 @@
+import { ErrInvariantFailed } from "./errors";
```

Comment:
Can we avoid having `utils`? We have it in erdjs, but in new repos where we could start clean we can group these types of functions better

Reply:
Fixed, renamed to `assertions`.